master/internal/resourcemanagers/resource_pool.go

```diff
@@ -377,15 +378,19 @@ func (rp *ResourcePool) receiveAgentMsg(ctx *actor.Context) error {
 
 func (rp *ResourcePool) moveJob(msg job.MoveJob) error {
 	// find out what is the job before or after the anchor
-	queueInfo := rp.scheduler.JobQInfo(rp)
+	// queueInfo := rp.scheduler.JobQInfo(rp)
 
-	newPos, err := computeNewJobPos(msg, queueInfo, rp.queuePositions)
+	fmt.Println(rp.config.PoolName, "moveJob: ", msg)
```

Comment:
these would be deleted before we release them. It's been helping with debugging. logs would work too, maybe better

Reply:
actually, we don't have access to ctx.Log or our master logger here unless we add it to the fn signature. since this is temporary wdy think about just keeping it as is?